tools/stress-cluster/cluster/kubernetes/stress-test-addons/templates/_util.tpl

```diff
@@ -27,16 +27,19 @@ Fields added to global context and returned:
 .Stress.Scenario - A `.` value passed down from a range loop over a scenarios list
             from values.yaml or a default value "stress".
 .Stress.ResourceGroupName - A pre-calculated resource group name value that can be passed down to various configurations that require it.
+.Stress.BaseName - A six character unique id derived from .Stress.ResourceGroupName that can be used in contexts with strict naming restrictions.
```

Comment:
Isn't BaseName randomly generated (L40) and not derived from .Stress.ResourceGroupName

Reply:
Whoops. I forgot to update this comment after my first iteration that derived it.